Feature - remove API Management defaults script (#51)
arcus-azure/arcus.scripting: docs/preview/features/powershell/azure-api-management.md

> | `ServiceName`       | yes       | The name of the API Management service located in Azure                                                  |

Reviewer: ```suggestion
| `ServiceName`       | yes       | The name of the Azure API Management instance located in Azure                                                  |
```

> | `ApiId`             | yes       | The ID to identify the API running in API Management                                                     |

Reviewer: ```suggestion
| `ApiId`             | yes       | The ID to identify the API running in Azure API Management                                                     |
```

# New API operation '$OperationName' on API Management service was added.
```	

> ## Remove all API Management defaults from the service

Reviewer: It's a product
```suggestion
## Remove all Azure API Management defaults from the service
```

> Remove all default API's and products from the API Management service ('echo-api' API, 'starter' & 'unlimited' products), including the subscriptions.

Reviewer: It's a product
```suggestion
Remove all default API's and products from the Azure API Management service ('echo-api' API, 'starter' & 'unlimited' products), including the subscriptions.
```

| Parameter       | Mandatory | Description                                              |
| --------------- | --------- | -------------------------------------------------------- |

> | `ResourceGroup` | yes       | The resource group containing the API Management service |

Reviewer: It's a product
```suggestion
| `ResourceGroup` | yes       | The resource group containing the Azure API Management service |
```

> | `ServiceName`   | yes       | The name of the API Management service located in Azure  |

Reviewer: It's a product
```suggestion
| `ServiceName`   | yes       | The name of the Azure API Management service located in Azure  |
```

> | `ServiceName`   | yes       | The name of the Azure API Management service located in Azure  |

Reviewer: Reason why I'm renaming all of them is that we are referring to the instance of the service, not the service itself #nitpicking
```suggestion
| `ServiceName`   | yes       | The name of the Azure API Management instance located in Azure  |
```

Reviewer: This makes me wonder, if all open APIM PRs are merged if we should rename all `ServiceName` parameters to `InstanceName`; but just a suggestion. Can be confusing as well if it's multi-instance setup.

Up to @mbraekman

Reviewer 2: Good point, agree with this one.
#62 created for this.

```powershell
PS> Remove-AzApiManagementDefaults -ResourceGroup $ResourceGroup -ServiceName $ServiceName
# Removing Echo Api...
# Removing Starter product...
# Removing Unlimited product...
```

> ## Import policy to an API in the API Management service

Reviewer: It's a product
```suggestion
## Import policy to an API in the Azure API Management service
```

Imports a base-policy from a file to an API in Azure API Management.

| Parameter        | Mandatory | Description                                               |
| ---------------- | --------- | --------------------------------------------------------- |

> | `ResourceGroup`  | yes       | The resource group containing the API Management service  |

Reviewer: It's a product
```suggestion
| `ResourceGroup`  | yes       | The resource group containing the Azure API Management service  |
```

> | `ServiceName`    | yes       | The name of the API Management service located in Azure   |

Reviewer: It's a product
```suggestion
| `ServiceName`    | yes       | The name of the Azure API Management instance |
```

> | `ApiId`          | yes       | The ID to identify the API running in API Management      |

Reviewer: It's a product
```suggestion
| `ApiId`          | yes       | The ID to identify the API running in Azure API Management      |
```

## Import policy to an operation in the API Management service
Imports a policy from a file to an API operation in Azure API Management.

| Parameter        | Mandatory | Description                                                     |
| ---------------- | --------- | --------------------------------------------------------------- |

> | `ResourceGroup`  | yes       | The resource group containing the API Management service        |

Reviewer: It's a product
```suggestion
| `ResourceGroup`  | yes       | The resource group containing the Azure API Management instance |
```

> | `ServiceName`    | yes       | The name of the API Management service located in Azure         |

Reviewer: It's a product
```suggestion
| `ServiceName`    | yes       | The name of the Azure API Management service instance |
```

> | `ApiId`          | yes       | The ID to identify the API running in API Management            |

Reviewer: It's a product
```suggestion
| `ApiId`          | yes       | The ID to identify the API running in Azure API Management            |
```
